frontend/src/utils/cellRenderers.jsx

```diff
@@ -52,8 +53,8 @@ export const StatusRenderer = (props) => {
 
 export const OrgStatusRenderer = (props) => {
   const location = useLocation()
-  const statusArr = ['Registered', 'Unregistered', 'Suspended', 'Canceled']
-  const statusColorArr = ['success', 'info', 'warning', 'error']
+  const statusArr = Object.values(ORGANIZATION_STATUS)
```

Comment:
use the previously mentioned function
const statusArr = getAllOrganizationStatuses();